pinpoint-flink/build/profiles/release/pinpoint-web.properties

```diff
@@ -0,0 +1 @@
+pinpoint.zookeeper.address=#PINPOINT_ZOOKEEPER_ADDRESS#
```

Comment:
I've tried to not put it and in that case we use the one include in `pinpoint-flink-job.jar` which in some way override the other value, may be if we remove them from that jar it will work I didn't tri it yet.